Tests/CommandsTests/PackageToolTests.swift

```diff
@@ -172,6 +172,52 @@ final class PackageToolTests: XCTestCase {
         }
     }
 
+    func testInitCustomNameEmpty() throws {
+        mktmpdir { tmpPath in
+            let fs = localFileSystem
+            let path = tmpPath.appending(component: "Foo")
+            try fs.createDirectory(path)
+            _ = try execute(["-C", path.asString, "init", "--name", "CustomName", "--type", "empty"])
+            XCTAssert(fs.exists(path.appending(component: "Package.swift")))
+            XCTAssertEqual(try fs.getDirectoryContents(path.appending(component: "Sources")), [])
+            XCTAssertEqual(try fs.getDirectoryContents(path.appending(component: "Tests")), [])
+        }
+    }
+    
+    func testInitCustomNameExecutable() throws {
```

Comment:
Let's just keep this test and remove the other two. These test are expensive to run and one is enough to ensure it will work.

Reply:
Sorry. Fixed